src/main/java/net/greghaines/jesque/worker/JedisFactory.java

```diff
@@ -0,0 +1,22 @@
+package net.greghaines.jesque.worker;
+
+import net.greghaines.jesque.Config;
+import redis.clients.jedis.Jedis;
+
+/**
+ * Created by dimav on 17/08/17.
+ *
+ */
+public class JedisFactory {
```

Comment:
I needed helper class that will allow me to create jedis for 2 threads. As the second thread will run in redis, I don't need it anymore